conf/config.yaml

```diff
@@ -1,12 +1,44 @@
 # Hydra defaults and settings. Ignore these (don't change)
 defaults:
   - _self_
-  - date_ranges: date_ranges
-  
   - paths: default
 
 report:
+  slack_channel: "#semif-datareports"
+
 pipeline:
+  - list_blob_contents
   - local_batch_table_generator
-#folders and their corresponding ile types that will be matched in the data directory
-matching_folders: {'images':'jpg','instance_masks':'png','semantic_masks':'png','metadata':'json'}
\ No newline at end of file
+  - report
+
+list_blob_contents:
+  unprocessed_folders:
+    - images
+    - autosfm
+    - metadata
+    - meta_masks
+  processed_folders:
+    - autosfm
+    - metadata
+    - meta_masks
+  file_extensions:
+    images: [ '.jpg' ]
+    raw_images: [ '.arw', '.raw' ]
+    metadata: [ '.json' ]
+    meta_masks/semantic_masks: [ '.png' ]
+  batch_prefixes:
+    - MD
+    - TX
+    - NC
+
+local_batch_generator:
+  bbot_versions:
+    MD:
+      v2.0: [ "2022-03-04", "2024-05-16" ]
+      v3.0: [ "2024-06-24", "2025-03-14" ]
+    NC:
+      v2.0: [ "2022-03-04", "2024-03-14" ]
+      v3.0: [ "2024-07-08", "2024-10-31" ]
+      v3.1: [ "2024-12-02", "2025-03-14" ]
```

Comment:
change the end of V3.1 using some form of datetime now. Here's an example from another repo:
```yaml
job_now_date: &nowdir ${now:%Y-%m-%d}
job_now_time: &nowtime ${now:%H:%M:%S}
```